activerecord/lib/active_record/schema_dumper.rb

```diff
@@ -29,6 +29,12 @@ class SchemaDumper #:nodoc:
     # should not be dumped to db/schema.rb.
     cattr_accessor :chk_ignore_pattern, default: /^chk_rails_[0-9a-f]{10}$/
 
+    ##
+    # :singleton-method:
+    # Specify a custom regular expression matching exclusion constraints which name
+    # should not be dumped to db/schema.rb.
+    cattr_accessor :excl_ignore_pattern, default: /^excl_rails_[0-9a-f]{10}$/
```

Comment:
probably a silly question, but why wouldn't you want exclusion constraints dumped to schema.rb?

Reply:
We definitely want them dumped (and they are), what we don't want to include is the default Rails-generated constraint name ([caller of this class attribute](https://github.com/rails/rails/pull/40224/files#diff-22750093c2231cef8c58e7e94a0a9266267d33bd990c607475e55fe285fccb8aR263-R265)).